### ISSUE TITLE
More Centipede Lag!

### ISSUE THREAD

**Issue by 400Ferrets** [sam-k0/LC_FixCentipedeLag#3]
if you jump in a pit with a centipede on your head it will spam the log with
`[Error  : Unity Log] "CalculatePolygonPath" can only be called on an active agent that has been placed on a NavMesh.`
or
`[Error  : Unity Log] "SetDestination" can only be called on an active agent that has been placed on a NavMesh.`
causing horrific stuttering
I believe this is a different centipede lag than the one the mod fixes, but it would be neat if you fixed this one too.

**Comment by sam-k0** [comment]
Yeah, you're right. I didn't update the mod for the latest game version yet. I will look into this though.
